test/jdk/java/awt/font/TextLayout/RotFontBoundsTest.java

```diff
@@ -60,10 +60,10 @@ public final class RotFontBoundsTest {
     private static final String TEXT = ".This is a STRINg.";
 
     private static final String INSTRUCTIONS =
-            "A string \u201C" + TEXT + "\u201D is drawn at eight different "
```

Comment:
I really don't like these tests being changed. It isn't part of the JDK build.
People compile these in all sorts of locales.
Please revert all the changes in the client tests.

Reply:
I'd like this reverted.
People *build* these tests in their local repos using javac and whatever their locale is.